src/components/ThankYouPage/ThankYouPage.js

```diff
@@ -31,6 +32,7 @@ const ThankYouPage = ({ onSuccess, t }) => {
   };
   const paymentMethodName = readablePaymentMethod[paymentMethod];
   const currencySymbol = currencyFormat[currency];
+  const formattedTotalAmount = formatNumber(totalAmount);
```

Comment:
same here - we want to keep the same variable name in translation: totalAmount. You can rename it when getting from store and then here you may have something like: 
```
const totalAmount = formatNumber(totalAmountFromStore);
```